app/src/main/java/org/mozilla/fenix/components/metrics/InstallationPing.kt

```diff
@@ -75,6 +75,7 @@ class InstallationPing(private val context: Context) {
      * */
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     internal fun checkMetricsNotEmpty(): Boolean = listOf(
+        context.settings().adjustCampaignId,
```

Comment:
So the `adjustCampaignId` is going to exist, regardless of whether there was a campaign or it was organic, correct? Could we either add a comment, or update the installation ping documentation to include this?

Reply:
That's correct! I will update the documentation.

Reply:
Oh, I already had that in the docs: 

> If the install is organic, this will be empty.